package.json

```diff
@@ -201,6 +202,13 @@
           "group": "display@6",
           "when": "webviewId === memory-inspector.memory"
         }
+      ],
+      "editor/title": [
+        {
+          "command": "memory-inspector.show",
+          "group": "navigation",
+          "when": "memory-inspector.canRead && (resourceLangId === c || resourceLangId === cpp)"
+        }
```

Comment:
You are correct, it is not strictly part of this PR and so I only mentioned it as a "minor" in the PR description

> Add title toolbar item for C/C++ file to access open memory inspector

Mainly because I used it a lot to open a memory inspector from a source file. I understand that the condition is far from complete in this regard but since we are only opening an empty memory inspector, I don't see the harm. However, I understand if you want me to move this out of this PR as wel.

Reply:
Agree that this is one we should address together with other entry points proposed in #52 in separate PRs.
I believe there was a discussion around source editors already.